### PR TITLE
humidifier card: fix humidity not visible

### PR DESCRIPTION
**Pull request by Shulyaka**
## Proposed change

Sometimes I have a strange glitch on my humidifier card: the target humidity value is not shown, until I switch to another lovelace tab and back, or until I move the slider. This small fix rectifies the glitch.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/cards/hui-humidifier-card.ts [hunk 1]
```diff
@@ -84,6 +84,10 @@ export class HuiHumidifierCard extends LitElement implements LovelaceCard {
       `;
     }
 
+    if (!this._setHum) {
+      this._setHum = this._getSetHum(stateObj);
+    }
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-humidifier-card.ts:89]: This is not the correct fix, we should not have side effects in render.

Can you find out when the humidity is not visible? Is it after a certain action?

**Comment by Shulyaka** [comment]
May I ask you to try the following steps to reproduce?
1. Create a new dashboard
2. Create a new view with 'panel' mode
3. Add a humidifier card to it
4. Save the edits
5. Reload the page (i.e. press F5)
6. Right after reload you might probably see it.

Sample yaml:
```yaml
views:
  - theme: Backend-selected
    path: ''
    title: Bedroom
    type: panel
    badges: []
    cards:
      - type: humidifier
        entity: humidifier.demo_humidifier
```

image

**Comment by Shulyaka** [comment]
It's even more obvious when the humidifier is on:

изображение

I lack the knowledge on the architecture to sort it out myself.

**Comment by Shulyaka** [comment]
Ok, second attempt.
I still don't know the root cause, but this also fixes the issue.
No side effects in the render function this time :)

**Review by bramkragten — APPROVED** [review]

**Pull request merged**